os/Kconfig

```diff
@@ -39,6 +39,18 @@ config DEFAULT_SMALL
 		have already been selected in your configuration file.  This applies
 		only to new settings that require a default value.
 
+config DOCKER_VERSION
+	string "Docker Image Version"
+	default "1.5.8"
+	---help---
+		This config indicates docker image version. To ensure compatibility with
+		different chipsets, we have developed distinct version of docker images tailored
+		to each arm-none-eabi toolchain required. This approach streamlines development
+		by providing specific environments that align with the chipset's needs.
+
+		NOTE: RTL8730E supports arm-node-eabi toolchain version 10.3.1. So, please ensure
+		docker version 2.0.0 for RTL8730E boards.
```

Comment:
@gSahitya-samsung @abhishek-samsung @r-prabu Could you check this location is proper or not? How about c++ exception for RTL8721CSM? With docker version 1.5.8, is it working on it? If not, do we need to mention like this? It looks it does not have a dependency with a chip.

Reply:
cpp exception need 2.0.0 on tp1x also. By default exceptions are not enabled on tp1x. But if enabled, using 1.5.8 will result in build error.